web/web.go

```diff
@@ -58,7 +58,7 @@ func getDefault(ctx *context.Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	http.Redirect(w, r,
-		rt.URL,
+		rt.URL+s,
```

Comment:
looks like `s` contains the leading slash... i don't think it should.
you might want to add after an `param=` at the end, but now I am getting `param=/foobar` which is not what I want.

Reply:
I have added `+1` to parseName function

https://github.com/mathieujobin/go/commit/e47e38e8e09d8ad7d2d78e9a7735849dc1c7cac0